source/docs/appendices/troubleshooting.md

```diff
@@ -0,0 +1,84 @@
+---
+id: troubleshooting
+title: Troubleshooting
+---
+
+Over the years, many developers have used Front-Commerce. A learning curve for such a solution exist, and we have detected recurring errors that may be difficult to troubleshoot for developers new to Front-Commerce or starting a new project.
+
+This page contains the most common errors you may encounter, along with information that will help to solve them quickly.
+
+## My server is not starting
+
+> I cannot prepare the application or make it start
+
+1. check the stdout/stderr of your server and the `/logs` folder, there may be useful information
+2. try to prefix your command with `DEBUG="front-commerce:scripts"` to enter verbose mode (e.g: `DEBUG="front-commerce:scripts" npm run start`)
```

Comment:
Because that's what I always do… to ensure special chars (`:`, `,` …) do not mess with the system I took this habit.

Here is an example of my local config:
```
DEBUG="axios,front-commerce:cache"
```

I think I already added quotes at several locations in the documentation. Let's decide whether we document both undifferently or we homogeneize this across the whole documentation, and do it at once before merging `develop` in `master`.